src/main/java/org/spongepowered/common/mixin/api/mcp/entity/EntityMixin_API.java

```diff
@@ -447,7 +447,7 @@ public boolean isLoaded() {
 
     @Override
     public void remove() {
-        this.isDead = true;
+        setDead();
```

Comment:
Just remember to always qualify field and method calls, so this should be
```suggestion
        this.setDead();
```